e2e-tests/cypress/tests/integration/channels/collapsed_reply_threads/following_spec.ts

```diff
@@ -254,7 +255,9 @@ function postMessageWithReply(channelId, postSender, postMessage, replySender, r
     });
 }
 
-function scrollThreadsListToEnd(maxScrolls = 1, scrolls = 0) {
+Cypress.Commands.add('postMessageWithReply', postMessageWithReply);
```

Comment:
Do we still need to add this to `Cypress.Commands` when we still call it directly above? I'd expect to only need this if and the `declare global` below if we changed the tests to call this with `cy.postMessageWithReply(...)`

Reply:
Also, I'm not sure how this works currently, but it looks like the `ChainableT` that this returns doesn't resolve to anything else, but the code that calls this above uses the return value of this. That feels like it shouldn't work, but I might be confusing Cypress's chainable objects with actual promise objects